lib/tasks/homepage.rake

```diff
@@ -10,5 +10,4 @@ namespace :homepage do
       Setting['feature.homepage.widgets.feeds.processes'] = true
     end
   end
-
 end
```

Comment:
Could you please add a trailing whitespace at the end of this file?

Reply:
Done